src/app/contextBroker/contextBroker.cpp

```diff
@@ -226,6 +227,7 @@ char            rush[256];
 #define DBUSER_DESC         "database user"
 #define DBPASSWORD_DESC     "database password"
 #define DB_DESC             "database name"
+#define TIMEOUT_DESC        "replica set timeout"
```

Comment:
What do you think about:

"timeout in seconds for connections to the replica set (ignored in the case of not using replica set)"

?

We need to introduce the "replica set" concept in the wording, as the timeout is only used in that case. If the user doesn't know what a replica set is, then this parameter is not for him/her :)